cabal-install/tests/IntegrationTests/exec/should_run/exit_with_failure_without_args.sh

```diff
@@ -0,0 +1,6 @@
+. ../common.sh
+
+# We should probably be using a .err file and should_fail,
```

Comment:
I'll create an issue to add it and assign it to myself. (No guarantees on when I'll get 'round to it, though.)